docs/.style/variables.sass

```diff
@@ -5,9 +5,16 @@ $drop-shadow: rgba(22, 22, 22, 0.2)
 $white: #FFF
 $pearl: #f6f7f7
 $menu-open: #347193
+$green: #92DC84
```

Comment:
There is no need to have the additional $green variable, just override $success similar to how we do it for $primary.